.claude/templates/hypothesis-ledger.md

```
@@ -0,0 +1,166 @@
+| CONFIRMED | `___` |
+| REJECTED | `___` |
+| REVISED | `___` |
+| Confirmation Rate | `___%` |
+
+---
+
+## Hypothesis Chain
+
+_Track how hypotheses relate to each other:_
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)